java-operators/test/com/baeldung/operators/ArithmeticTest.java

```diff
@@ -0,0 +1,106 @@
+package com.baeldung.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ArithmeticTest {
+
+	
+	@Test
+	public void arithmeticAddTiveTest() {

```

Comment:
There's a typo in the test name.

Reply:
done

Reply:
The correct English name is addition or sum.